sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/ReadFromKafkaDoFn.java

```diff
@@ -511,6 +529,9 @@ public ProcessContinuation processElement(
             }
           }
         }
+        LOG.error(
```

Comment:
warning seems more appropriate as it is more performance issue

Reply:
Done.